src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/attributes/DisplayAttributeInputs.tsx

```diff
@@ -73,7 +74,9 @@ const DisplayAttributeInputs = ({
             data-testid="ratio-input"
             onRatioChange={(newCQLRatio) => {
               setCurrentRatio(newCQLRatio);
-              setAttributeValue(newCQLRatio);
+              if (newCQLRatio.numerator.unit && newCQLRatio.denominator.unit) {
```

Comment:
this check is unnecessary

Reply:
<img width="758" alt="image" src="https://github.com/MeasureAuthoringTool/madie-patient/assets/47760891/a87253a3-bc2b-4936-91c3-c6439ee4ebc0">
Without it, this error happens

Reply:
this check should happen in the Quantity component itself. I don’t think there will be quantity without a unit. also, onChange should not be triggered until the quantities and ratio is valid